packages/docs/site/docs/main/guides/for-theme-developers.md

```diff
@@ -93,11 +93,11 @@ Some useful tools and resources provided by the Playground project to work with
 
 Through properties and [`steps`](/blueprints/steps) in the blueprint, you can configure the initial setup of your theme in the Playground instance.
 
-### Importing content for your demo
+### Providing content for your demo
 
 One of the things you may want to do to provide a good demo of your theme via Playground is to load default content to better highlight the features of your theme. This default content may include images or other assets.
```

Comment:
The sentence seems a bit long. See if this tightens it up a bit. 
"To provide a good demo of your theme via Playground, you may want to load it with default content that highlights the features of your theme. "

Reply:
Thanks for the suggestion! 

I have extracted the whole "Providing content for your demo" section into a new guide (see https://github.com/WordPress/wordpress-playground/pull/1747) to reference this content from both the "Playground for theme developers" and "Playground for plugin developers" guides.

I have updated this PR with your suggested shorter version in the info callout referencing the "Providing content for your demo" guide